docs/content/policy-reference.md

```diff
@@ -1316,10 +1316,14 @@ names, or dot-access style reference arguments:
 
 ```
 as
+contains
+data
 default
 else
 false
+if
 import
+input
```

Comment:
Should we add `in`, `every`.

Reply:
Yes, good call! Doesn't seem like `every` is treated as a keyword though, even in v1. 
This fails only on assigning to `in`
```rego
package play

import rego.v1

every := "foo"

in := "bar"
```
I think that's a bug?

Reply:
I would have expected consistent behavior. cc @johanfylling

Reply:
Yeah, that looks like a bug 😞.

Reply:
Alright, I’ll create a separate issue for that then, but will add in and every to the list in this PR.

Reply:
I'm fine with adding `in` and `every` either as part of this PR or as part of #6652.